### ISSUE TITLE
Need to consider config.apps.sample.php

### ISSUE THREAD

**Issue by mmattel** [owncloud/config-to-docs#1]
```
How to use
Just call following in your ownCloud core repo:
php path/to/convert.php
```
We need to consider that we have now two sample files to be processed
``config.sample.php``
``config.apps.sample.php``

**Comment by mmattel** [comment]
IMHO it is a must that both files are processed automatically and that there is a parameter if you want to do different.

**Comment by mmattel** [comment]
closing, fixed by #5
